src/Tracking/UsageTrackingOnBoarding.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace Give\Tracking;
+
+/**
+ * Class OnBoarding
+ * @package Give\Tracking
+ *
+ * This class uses to setup notice nag to website administrator if admin is not opt in for usage tracking and gives admin an option to directly opt-in.
+ *
+ * @since 2.10.0
+ */
+class UsageTrackingOnBoarding {
+	/**
+	 * Bootstrap
+	 *
+	 * @since 2.10.0
+	 */
+	public function boot() {
+		add_action( 'admin_notices', [ $this, 'addNotice' ] );
+	}
+
+	/**
+	 * Register notice.
+	 *
+	 * @sicne 2.10.0
+	 */
+	public function addNotice() {
+		$isAdminOptedIn = give_is_setting_enabled( give_get_option( AdminSettings::USAGE_TRACKING_OPTION_NAME, 'disabled' ) );
+
+		if ( $isAdminOptedIn || ! current_user_can( 'manage_give_settings' ) ) {
+			return;
+		}
+
+		$notice       = esc_html__( 'You can contribute to improve GiveWP. If you opt-in to "Usage Tracking" then we will track non-sensitive data of your website. We will use this information to improve plugin.', 'give' );
```

Comment:
Also, we'll want @DevinWalker and/or @mathetos to review the public facing message.

Reply:
@ravinderk see the way Gutenberg asks for permissions. Let's format it like that with the first bold sentence, italicized second sentence, and the "Yes" "No" buttons.

> **Can GiveWP collect data about the usage of the plugin?**
> *Usage data is completely anonymous, does not include any personal information, and will only be used to improve the software.*

Reply:
fixed: https://github.com/impress-org/givewp/pull/5460#issuecomment-730881805